packages/@ourworldindata/grapher/src/stackedCharts/StackedDiscreteBarChart.tsx

```diff
@@ -659,223 +656,100 @@ export class StackedDiscreteBarChart
+                    y={0}
+                    transform={`translate(${barX}, ${-barHeight / 2})`}
+                    width={barWidth}
+                    height={barHeight}
+                    fill={bar.color}
+                    opacity={isFaint ? 0.1 : 0.85}
+                    style={{
+                        transition: "height 200ms ease",
+                    }}
+                />
+                {showLabelInsideBar && (
+                    <text
+                        x={barX + barWidth / 2}
                         y={0}
-                        transform={`translate(${barX}, ${-barHeight / 2})`}
                         width={barWidth}
                         height={barHeight}
-                        fill={bar.color}
-                        opacity={isFaint ? 0.1 : 0.85}
-                        style={{
-                            transition: "height 200ms ease",
-                        }}
-                    />
-                    {showLabelInsideBar && (
-                        <text
-                            x={barX + barWidth / 2}
-                            y={0}
-                            width={barWidth}
-                            height={barHeight}
-                            fill={labelColor}
-                            opacity={isFaint ? 0 : 1}
-                            fontSize={labelFontSize}
-                            textAnchor="middle"
-                            dominantBaseline="middle"
-                        >
-                            {barLabel}
-                        </text>
-                    )}
-                </g>
-            </TippyIfInteractive>
+                        fill={labelColor}
+                        opacity={isFaint ? 0 : 1}
+                        fontSize={labelFontSize}
+                        textAnchor="middle"
+                        dominantBaseline="middle"
+                    >
+                        {barLabel}
+                    </text>
+                )}
+            </g>
         )
     }
 
-    private static Tooltip(props: TooltipProps): JSX.Element {
-        let hasTimeNotice = false
-        const { highlightedSeriesName, item } = props
-        const showTotal = !item.bars.some((bar) => bar.point.fake) // If some data is missing, don't calculate a total
-
-        const timeNoticeStyle = {
-            fontWeight: "normal",
-            color: "#707070",
-            fontSize: "0.8em",
-            whiteSpace: "nowrap",
-            paddingLeft: "8px",
-        } as React.CSSProperties
+    @computed private get Tooltip(): JSX.Element | undefined {
+        const {
+                tooltipState: { target, position, fading },
+                formatColumn: { unit, shortUnit },
+                manager: { endTime: targetTime },
+                inputTable: { timeColumn },
+            } = this,
+            item = this.placedItems.find(
+                ({ label }) => label == this.tooltipState.target?.entityName
```

Comment:
```suggestion
                ({ label }) => label == target?.entityName
```

Reply:
The problem is that that `this.base` has a different origin point than the parent's element so when we call `getRelativeMouse` it gives us a location whose y position doesn't include the height of the chart title. Note that if you make the suggested change the tooltip box is positioned much higher on the screen rather than vertically aligning the tooltip's entity name with the cursor position.

We could use `this.base` and hard-code a y-axis adjustment but using the parent's coordinate system felt better from a 'correctness' standpoint…

Reply:
This comment got misplaced perhaps?